storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -1928,6 +1928,12 @@ bool write_xtrabackup_info(MYSQL *connection) {
                  ") CHARACTER SET utf8 ENGINE=innodb",
                  false);
 
+  /* Upgrade from previous versions */
```

Comment:
if table is big is any performance impact to trigger this alter if binlog_pos is already TEXT? Just want to know if it is worth to execute this query only if column is not TEXT?

Reply:
@rahulmalik87 MySQL will detect that and won't do the `ALTER` . We tested it on the 5.7 version of the patch - https://github.com/percona/percona-xtrabackup/pull/1069#discussion_r614725099